lib/extend.es6

```diff
@@ -32,8 +31,17 @@ export default (options = {}) => {
     };
 };
 
+const api = new Api;
+let baseFileOpts;
 
 function handleExtendsNodes(tree, options) {
+    baseFileOpts = baseFileOpts || tree.options;
+    tree.options = tree.options || baseFileOpts;
+
+    if (!tree.messages) {
+        tree.messages = [];
+    }
```

Comment:
Do not need it. message always exists.

```diff
-    if (!tree.messages) {
-        tree.messages = [];
-    }
```

Reply:
see https://github.com/posthtml/posthtml-extend/pull/12#issuecomment-366657557